app/src/main/java/org/wikipedia/readinglist/ReadingListBehaviorsUtil.kt

```diff
@@ -135,6 +135,15 @@ object ReadingListBehaviorsUtil {
         }
     }
 
+    fun updateReadingListPage(activity: AppCompatActivity, item: ReadingListPage) {
+        activity.lifecycleScope.launch(exceptionHandler) {
```

Comment:
Does this really need to be bound to the lifecycleScope of the activity? This could be a case where `MainScope` is more appropriate.